tests/spread/integration/vim-tiny/task.yaml

```diff
@@ -0,0 +1,5 @@
+summary: Integration tests for vim-tiny
+
+execute: |
+  rootfs="$(install-slices vim-tiny_bins)"
+  chroot "${rootfs}/" vim.tiny --version
```

Comment:
We could salvage the better tests from https://github.com/canonical/chisel-releases/pull/301/files#diff-9daa776536c3c5e696dcdca0499bd27d45530a1cf8ac10cbb9fdd239e04e9855.